### PR TITLE
Add Mozilla Foundation strategy document to Foundation page

### PR DESCRIPTION
**Pull request by secretrobotron**
## Description
Add a link to the Mozilla Foundation strategy PDF to the Foundation page

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1292658

## Testing
Didn't add/change any tests. Just a text/url change. Needed?

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

1. Added link to document in side panel in base-resp.html.
2. Added link to document in body in index.html.

### CODE REVIEW

File: bedrock/foundation/templates/foundation/index.html [hunk 1]
```diff
@@ -32,6 +32,12 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
   {% endtrans %}
   </p>
 
+  <p>
+    {% trans url_open_strategy_document='https://mzl.la/foundation-strategy' %}
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/index.html]: You can isolate the text inside the link so all that surrounding HTML isn't part of the string. Then you can also just put the URL in directly and forego the trans block:

`<p><strong><a href="https://mzl.la/foundation-strategy" class="go">{{ _('Read the Mozilla Foundation’s 2016-2018 Strategy') }}</a></strong></p>`

Comment (secretrobotron) [inline bedrock/foundation/templates/foundation/index.html]: Oh, great! I copied what I found below. Does the `_(...)` handle translation?

**Comment by flodolo** [comment]
Foundation pages are not localized.

**Comment by craigcook** [comment]
r+ 🍓 

I squashed the two commits into one and slightly amended the commit message to reference the bug number. This was merged to master in https://github.com/mozilla/bedrock/commit/27486150e411e0b2107e4cd8159c6b50b4e32214

Thanks!

**Pull request closed without merging**